src/data/items.txt

```diff
@@ -4,7 +4,7 @@
 #	PRIMARY_USE:
 #	none,
 #	food, drink, spleen,
-#	potion, avatar
+#	beverage, potion, avatar
```

Comment:
I see that you add BEVERAGE to the usage comments in fullness.txt, but do not actually mark anything as a "beverage".